apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.spec.js

```diff
@@ -291,133 +291,4 @@ describe('DomaMetersImporter', () => {
         expect(errors).toEqual([['unknownUnitType'], ['unknownResource'], ['unknownIsAutomatic']])
     })
 
-    test('Transforms excel dates to format YYYY-MM-DD', () => {
-        // see packages/keystone/file/file-types/excel for more info about strange dates
-        const excelDate = '45532'
-        const expectedDate = '2024-08-28T00:00:00.000Z'
```

Comment:
Parser parses file as strings.
And DomaMetersImporter parses these strings to something
Before excep dates were parsed as strange numbers, like excep date variable here. And now there will be none of these values. It will be Date.toISOStrings here, so this test is outdated

Reply:
And test for case, when parser returns date string, already exists above in .spec.